Dockerfile

```diff
@@ -17,17 +17,23 @@ RUN gradle clean build --no-daemon -x test
 # Create a new Docker image with the Java runtime only
 FROM adoptopenjdk:11-jre-hotspot
 
+ARG offers_path
+ARG player_dataset_path
+ARG APP_PORT
+
+ENV APP_PORT=$APP_PORT
+
```

Comment:
@natalia-appcharge @AlexPolyakovsky  why do we need docker file here? Didnt we say that CD will be only in the mocker repo?

Reply:
yes, CD will be only in the mocker repo
But how publishers will build and run code from this repo?